rviz_default_plugins/CMakeLists.txt

```diff
@@ -543,6 +547,7 @@ if(BUILD_TESTING)
   if(TARGET point_cloud_transformers_test)
     target_include_directories(point_cloud_transformers_test PUBLIC ${TEST_INCLUDE_DIRS})
     target_link_libraries(point_cloud_transformers_test ${TEST_LINK_LIBRARIES})
+    ament_target_dependencies(marker_test ${TEST_TARGET_DEPENDENCIES})
```

Comment:
`marker_test` -> `point_cloud_transformer_test`

Reply:
Good catch, addressed in 0ee8721a7c0ca9baf6f2f654b950b70354b8adf9.